### DISCUSSION TITLE
The problem while using SearchPublicChats

### DISCUSSION THREAD

**Discussion by qing-wang** [egramtel/tdsharp#130] [Q&A] [answered]
I am building a client which utilizes tdsharp. However, I have a problem while using SearchPublicChats.

The result returned by the API does not match to "Global Search" result shows on iPhone Official Telegram App.

For instance, by passing "# tesla" as the argument of SearchPublicChats, the result returned by API contains only two chats. However,  by entering"# tesla" in search textfield of app, the result list contains more records.

**Comment by ForNeVeR** [comment, marked as answer]
Unfortunately, in this project, we merely provide bindings for the tdlib API, and cannot provide all the answers about how this API works internally (since it is server-side).

You may [try asking in the upstream repository](https://github.com/tdlib/td/issues).

**UPD:** the author has asked in https://github.com/tdlib/td/issues/2032.

**Reply by qing-wang** [reply]
Ok. Thanks for you reply.
